### PR TITLE
[15.0][ADD] account_analytic_security

### PR DESCRIPTION
**Pull request by IriaAlonso**
- Add new group "Create / Delete analytic account" 
- Add two new registration rules "Enabled creation/deletion all analytic accounts" and "Disabled creation/deletion all analytic accounts" 
- That allows to restrict something that had no restrictions, and that is to create or delete analytical accounts.
@dalonsod

### CODE REVIEW

File: account_analytic_security/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "name": "Account Analityc Security",
+    "summary": """
+        Add a new group and two registration rules to restrict users from creating or deleting analytical accounts
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "15.0.1.0.0",
+    'category': "Accounting & Finance",
+    "website": "https://github.com/solvosci/slv-account",
+    "depends": ["account"],
```
Comment (dalonsod) [inline account_analytic_security/__manifest__.py]: Is this dependency the minor one that must be required? Please check

Comment (IriaAlonso) [reply account_analytic_security/__manifest__.py]: depending of analytics also works but this warning appears: 
imagen

**Comment by IriaAlonso** [comment]
@dalonsod changes done :+1:

File: account_analytic_security/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: account_creation_analytic_account
+#: model:res.groups,name:account_creation_analytic_account.group_creation_analytic_account
+msgid "Create / Delete analytic account"
+msgstr "Crear / Eliminar cuentas analiticas"
```
Comment (dalonsod) [inline account_analytic_security/i18n/es.po]: Missing accent:
```suggestion
msgstr "Crear / Eliminar cuentas analíticas"
```

File: account_analytic_security/security/account_security.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data noupdate="0">
```
Comment (dalonsod) [inline account_analytic_security/security/account_security.xml]: Make proper indentation (`<data>` nodes are inside `<odoo>` one)

**Review by dalonsod — CHANGES_REQUESTED** [review]
Attend minor comments and I'll test it

**Comment by IriaAlonso** [comment]
@dalonsod done

**Pull request merged**